<launch>
  <arg name="planning_group" default="right_arm"/>
  <arg name="metrics_enabled" default="True"/>
  
  <group ns="baxter">
    <!-- Upload robot and semantic description that corresponds to the URDF -->
    <!-- Send urdf to param server -->
    <param name="robot_description" textfile="$(find baxter_openrave)/urdf/baxter.urdf" />
    <!-- the semantic description that corresponds to the URDF -->
    <param name="robot_description_semantic" textfile="$(find baxter_moveit_config)/config/baxter.srdf" />
    
    <!-- Start the kinematics services node -->
    <node name="kinematics_services" pkg="moveit_kinematics_interface" type="kinematics_services" output="screen">
      <rosparam command="load" file="$(find baxter_moveit_config)/config/kinematics.yaml"/>
      <param name="metrics_enabled" value="$(arg metrics_enabled)"/>
      <param name="planning_group" value="$(arg planning_group)"/>
      <param name="ik_dynamic_reconfigure" value="false"/>
    </node>
  </group>
  
</launch>
